observatory-platform/observatory/platform/utils/proc_utils.py

```diff
@@ -52,3 +56,21 @@ def stream_process(proc: Popen, debug: bool) -> Tuple[str, str]:
         if proc.poll() is not None:
             break
     return output_concat, error_concat
+
+
+def run_bash_cmd(cmd: str):
+    """Run a command in the bash shell and wait until it's done.  Log the output.
+    Raise an exception where there's stderr.
+
+    :param cmd: Command to run.
+    """
+
+    p = Popen(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE, executable="/bin/bash")
```

Comment:
Maybe add `shell=True` as a keyword argument in run_bash_cmd so that it can be turned off if required.